kong/dao/db/cassandra.lua

```diff
@@ -322,7 +322,11 @@ local function serialize_arg(field, value)
   elseif field.type == "timestamp" then
     return cassandra.timestamp(value)
   elseif field.type == "boolean" then
-    return cassandra.boolean(value)
+    if type(value) == "boolean" then
+      return cassandra.boolean(value)
+    else
+      return cassandra.boolean(value == 'true')
```

Comment:
style: we should stick to double quotes and follow the early-exit paradigm we try to enforce for short if/else branches. 
```lua
if type(value) == "boolean" then
  return cassandra.boolean(value)
end

return cassandra.boolean(value == "true")
```

It is fine to have this fix here considering this is legacy code, but part of the reason why this doesn't really belong here (cc @salazar) is because when given a value like `bool_attr="hello"`, we will return rows with `bool_attr=false`, instead of returning HTTP 400, which can be surprising.

Reply:
This is a fair assessment, I wasn't sure where we wanted to stand with strings that aren't equal to their counterpart so took the safest approach, I was also unsure about whether we wanted to support another paradigm you see in boolean based query strings values such as using `1` and `0`.